src/cli/buckets_config.rs

```diff
@@ -99,3 +114,66 @@ fn buckets(
 
     Ok(converted.into_pipeline_data())
 }
+
+fn get_server_bucket_config(
+    cluster: &RemoteCluster,
+    name: String,
+    ctrl_c: Arc<AtomicBool>,
+    span: Span,
+) -> Result<HttpResponse, ShellError> {
+    cluster
+        .cluster()
+        .http_client()
+        .management_request(
+            ManagementRequest::GetBucket { name },
+            Instant::now().add(cluster.timeouts().management_timeout()),
+            ctrl_c,
+        )
+        .map_err(|e| client_error_to_shell_error(e, span))
+}
+
+fn get_capella_bucket_config(
+    org: &RemoteCapellaOrganization,
+    project: String,
+    cluster: &RemoteCluster,
+    bucket: String,
+    ctrl_c: Arc<AtomicBool>,
+    span: Span,
+) -> Result<HttpResponse, ShellError> {
+    let client = org.client();
+    let deadline = Instant::now().add(org.timeout());
+
+    let org_id = find_org_id(ctrl_c.clone(), &client, deadline, span)?;
+    let project_id = find_project_id(
+        ctrl_c.clone(),
+        project,
+        &client,
+        deadline,
+        span,
+        org_id.clone(),
+    )?;
+
+    let cluster_id = find_cluster_id(
+        cluster.display_name().unwrap_or("".to_string()),
+        ctrl_c.clone(),
+        cluster.hostnames().clone(),
+        &client,
+        deadline,
+        span,
+        org_id.clone(),
+        project_id.clone(),
+    )?;
+
+    client
+        .capella_request(
+            CapellaRequest::GetBucketV4 {
```

Comment:
I don't think this returns the bucket config? This command was intended to be quite low level and return the config as reported by /pools/default/b/<bucket>

Reply:
For some reason I thought that this endpoint wouldn't work for Capella clusters, so was just doing a get that printed out all the fields we get. But it does work so have changed it back